INSTALL.md

```diff
@@ -146,6 +293,38 @@ If running on OS X you should make sure that the Xcode and the Xcode command lin
 
 If you are using Postgres.app on OSX, a database user will be created with your system username. If you want to use this user account, change the `USER` and `ADMIN_USER` settings to your username under the `[postgres]` section of your Qiita config file.
 
+### conda
```

Comment:
Again, I am hesitant on supporting installation instructions for packages that we don't support at all - those can change without us noticing and these instructions get outdated. Point to the troubleshooting conda official instructions.